src/System.ServiceModel.Syndication/tests/System.ServiceModel.Syndication.Tests.csproj

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netstandard-Release|AnyCPU'" />
   <ItemGroup>
-    <Compile Include="$(MsBuildThisFileDirectory)**\*.cs" Exclude="$(MsBuildThisFileDirectory)async\**\*.cs" />
```

Comment:
Do you actually need two test projects? Generally we have one, with apprporiate conditions on $(TargetGroup), and in the tests\Configurations.props you would have netstandard and netcoreapp, and each would run as appropriate. That's less clumsy than two projects

Reply:
Good point. Thanks for the suggestions. I'll change this.